api/nodes/serializers.py

```diff
@@ -212,6 +206,7 @@ def profile_image_url(self, user):
 
     class Meta:
         type_ = 'contributors'
+        target_type_ = 'users'
```

Comment:
After our discussion, I will make target_type_ an argument to the field constructor. That is cleaner and will allow for creating multiple relationships in the future.